### PR TITLE
updates for M104

### PR DESCRIPTION
**Pull request by hostirosti**
Fixes missing package + fixes oldHeaderSwitch function

### CODE REVIEW

**Comment by hostirosti** [comment]
@apasel422 PTAL

File: conversion-measurement/functions/apps/adtech.js [hunk 1]
```diff
@@ -358,4 +358,4 @@ adtech.post(
   }
 )
 
-exports.adtech = functions.https.onRequest(adtech);
+exports.adtech = functions.https.onRequest(adtech);
```
Comment (apasel422) [inline conversion-measurement/functions/apps/adtech.js:361]: nit: Maybe restore the newline at the end of the file?

File: conversion-measurement/functions/apps/adtech.js [hunk 2]
```diff
@@ -251,8 +251,8 @@ adtech.get('/conversion', (req, res) => {
 
   // Debug report (common to event-level and aggregate)
   console.log("Conversion Cookies Set: ", req.cookies)
-  // Optional: set a debug key, and give it the value of the legacy measurement 3P cookie.
-  // This is a simple approach for demo purposes. In a real system, you would still make this key a unique ID, but you may map it to additional trigger-time information that you deem useful for debugging or performance comparison.
+    // Optional: set a debug key, and give it the value of the legacy measurement 3P cookie.
+    // This is a simple approach for demo purposes. In a real system, you would still make this key a unique ID, but you may map it to additional trigger-time information that you deem useful for debugging or performance comparison.
```
Comment (apasel422) [inline conversion-measurement/functions/apps/adtech.js:255]: nit: Are these whitespace changes necessary?

**Review by apasel422 — APPROVED** [review]

File: conversion-measurement/functions/apps/adtech.js (hunk 2 above)
Comment (hostirosti) [reply conversion-measurement/functions/apps/adtech.js:255]: They are not, seems visual code introduced them, I can remove them.

**Comment by hostirosti** [comment]
@apasel422 addressed your comments.

**Review by apasel422 — APPROVED** [review]

**Pull request merged**